python-package/lightgbm/basic.py

```diff
@@ -3516,6 +3530,31 @@ def refit(self, data, label, decay_rate=0.9, **kwargs):
         decay_rate : float, optional (default=0.9)
             Decay rate of refit,
             will use ``leaf_output = decay_rate * old_leaf_output + (1.0 - decay_rate) * new_leaf_output`` to refit trees.
+        reference : Dataset or None, optional (default=None)
+            reference for ``data``.
+        weight : list, numpy 1-D array, pandas Series or None, optional (default=None)
+            Weight for each ``data`` instance. Weight should be non-negative values because the Hessian
+            value multiplied by weight is supposed to be non-negative.
+        group : list, numpy 1-D array, pandas Series or None, optional (default=None)
+            Group/query size for ``data``.
+        init_score : list, numpy 1-D array, pandas Series or None, optional (default=None)
+            Init score for ``data``.
+        feature_name : list of strings or 'auto', optional (default="auto")
```

Comment:
We use the name of real Python type to reference string type in our docstrings, refer to #4557.

```suggestion
        feature_name : list of str, or 'auto', optional (default="auto")
```

Reply:
got it